src/screens/Settings/NotificationSettings.tsx

```diff
@@ -72,7 +72,7 @@ export function NotificationSettingsScreen({}: Props) {
                 <Toggle.Item
                   name="enabled"
                   label={_(msg`Enable priority notifications`)}
-                  style={[a.flex_1, a.justify_between]}>
+                  style={[a.flex_1, a.justify_between, a.pt_sm]}>
```

Comment:
is this a residual change from when this setting was in this category?

Reply:
ah oops, yeah :p

Fixed